README.md

```diff
@@ -1,77 +1,90 @@
+You can also have sequences to add manually, via the filenames `manual_seqs` and `manual_meta` in the appropriate `vp1` or `genome` folder. These must also be formatted and already checked that they are either full-genome or contain VP1 (depending on the run). Data replacing or extending the 'Swedish' data in the top-level `data` folder should be full-genome, as it will be used in both full-genome and VP1 runs.
 
 You can add some extra data, particularly for sequences from GenBank where more detailed information can be scraped from papers, using the file named by `extra_meta`. This file should be tab-delimited with five columns: accession, date, age, sex, symptom, country, region. The data will be matched up to the combined metadata (from all of the above files) by accession. Dates need to be in the format YYYY-MM-DD and sex as `M` or `F`. Age can be multiple formats, such as: 2y3m, 20m, 2y, 15d, <12y, >79y, .5-10y. Unless specified by 'm' or 'd', numbers are taken as years: 2 is 2 years and 0.3 is 0.3 years. Ranges and greater-/less-than can be in months or years: 10m-3y. Symptoms can be as you wish, but should be consistent ('afm' and 'AFM' will be two different things).
 
 # Technical Notes
 
 ## Strain names
-In ViPR downloads as specified above, `strain` is not a unique identifier, as multiple segments may come from the same `strain`. This causes problems unique to VP1 analysis (with full-genome, this is not an issue). To handle this, in the VP1 run, the `vipr_parse.py` script generates new `strain` identifiers by combiing the original `strain` column with the accession number, separated by a double-underscore. 
+In downloads as specified above, `strain` is not a unique identifier, as multiple segments may come from the same `strain`. This causes problems unique to VP1 analysis (with full-genome, this is not an issue). To handle this, in the VP1 run, the `vipr_parse.py` script generates new `strain` identifiers by combiing the original `strain` column with the accession number, separated by a double-underscore. 
 
 Unlike the VP1 run, strain names are not modified during the full-genome run.
 
 ## Blasting
-ViPR sequences are not reliably labelled with the segment(s) they include (excepting whole-genome, it seems). In order to decide which sequences contain VP1, this script creates a local BLAST database against an EV-D68 reference genome VP1 sequence, then BLASTs all downloaded sequences against it. 
+Sequences from NCBI are not reliably labelled with the segment(s) they include (excepting whole-genome, it seems). In order to decide which sequences contain VP1, this script creates a local BLAST database against an EV-D68 reference genome VP1 sequence, then BLASTs all downloaded sequences against it. 
```

Comment:
I believe, in other pathogen repos, we accomplish this same thing by using `augur align` with a reference sequence that's just the gene of interest, which (effectively) truncates the resulting MSA — might be worth considering that, as it will remove the need to do the BLAST install locally...